### ISSUE TITLE
Error: Failed to resolve extends for '\PHPUnit\Framework\TestCase' in (DateTime,DateTimeImmutable, ...)

### ISSUE THREAD

**Issue by Gert-dev** [twifty/atom-phpunit-integrator#44]
This seems to appear when trying to run tests in the file via `ctrl-alt-f`:

```
php-unit-test-runner.js? [sm]:303
        at PhpCoreService.loadCache (php-core-service.js? [sm]:133)
        at PhpCoreService.<anonymous> (php-core-service.js? [sm]:30)
        at Generator.next (<anonymous>)
        at step (/home/werk/.atom/packages/php-unit-integrator/lib/proxy/php-core-service.js:9)
        at <anonymous>
```

**Comment by twifty** [comment]
Did you miss the error (first line) there?

**Comment by twifty** [comment]
Ahh, found it. You are extending `\PHPUnit\Framework\TestCase` without using a `use` statement, right?

I'll add a check for that. I also noticed an uncaught exception in the console while editing the class `extends`.

**Comment by Gert-dev** [comment]
Absolutely correct, it is indeed an FQCN without a use statement.

**Comment by twifty** [comment]
I neglected to check for (un)qualified classnames and instead just checked the list of use aliases. I'll push these fixes in a few minutes.

**Comment by twifty** [comment]
Fixed in 0.0.49
